ccmlib/scylla_repository.py

```diff
@@ -122,3 +126,10 @@ def __get_dir():
     if not os.path.exists(repo):
         os.mkdir(repo)
     return repo
+
+
+def run_scylla_install_script(install_dir, target_dir):
+    subprocess.check_call(['bash', '-c', ''' sed 's|"$prefix|"$root/$prefix|' -i install.sh '''], cwd=install_dir)
```

Comment:
This is non-maintainable. Please open an issue against scylla for fixing this in `install.sh`. Does it need another option or is this valid at all times?

Reply:
I opened scylladb/scylla#4949

Reply:
I'll remove it, once it's merged into master